application/F3DStarter.cxx

```diff
@@ -138,29 +138,40 @@ int F3DStarter::Start(int argc, char** argv)
     f3d::window& window = this->Internals->Engine->getWindow();
     f3d::interactor& interactor = this->Internals->Engine->getInteractor();
 
+    const auto loadFile = [this](int index, bool restoreCamera = false) -> bool
```

Comment:
since the lambda is used only in the other lambda, you can move it there to reduce the scope.